boards/sparkfun/pro_micro_rp2040/support/openocd.cfg

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2022 Tokita, Hiroshi <tokita.hiroshi@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+# Checking and set 'adapter speed'.
+# Set adapter speed that assigned by argument if not be seted.
```

Comment:
Worth making a pass on the grammar/spelling (and apply the same changes wherever else this was, apparently, copied from)